packages/kbn-optimizer/src/report_optimizer_stats.ts

```diff
@@ -35,7 +39,23 @@ export function reportOptimizerStats(reporter: CiStatsReporter, name: string) {
         }
 
         if (n.kind === 'C' && lastState) {
-          await reporter.metric('@kbn/optimizer build time', name, lastState.durSec);
+          await reporter.metrics([
+            {
+              name: '@kbn/optimizer build time',
+              subName: name,
+              value: lastState.durSec,
```

Comment:
I'm going to remove this metric because it's not really a measurement that is reliable, it's something that changes continuously based on what's happening in the build environment. Additionally, I'm going to rename `name` to `group` and `subName` to `metricId` to better indicate the intention behind the two values.